parameter_files/fates_params_default.cdl

```diff
@@ -1490,7 +1491,7 @@ data:
 
  fates_leaf_theta_cj_c4 = 0.999 ;
 
- fates_maintresp_model = 1 ;
+ fates_maintresp_model = 2 ;
```

Comment:
Also, what is our staging for making the change in the default file?  My sense from discussion with others is that the Atkin model is the path forward.  However, all of the other parameters are calibrated around the Ryan model, and this will undoubtedly have an impact on output.

Was the plan to switch over to Atkins first, and accept the differences good or bad to output, and then continue with other calibrations? Or was the plan to switch over to Atkins during a more comprehensive calibration exercise perhaps with the satphen simuluation or other biogeographically constrained testbed?

In either situations, we should probably run this through the ilamb mill to get a sense of the changes.

Reply:
I think for staging purposes, we should bring in the code first with this switch reverted to the current value, and if we decide to make it the default model, that would likely need to be accompanied by other default parameter changes as well.